_data/tool_and_resource_list.yml

```
@@ -2879,10 +2893,15 @@
 - description: NIH research program with diverse patient health, genetic, and lifestyle data.
   id: all-of-us
   name: All of Us
+  registry:
+    tess: All of Us
   url: https://www.researchallofus.org/
 - description: Large-scale biomedical database with genetic, clinical, and lifestyle data from 500,000 UK participants.
```

```suggestion
- description: Large-scale biomedical database with genetic, clinical, and lifestyle data from UK participants.
```

This is not a sustainable description